README.md

```diff
@@ -54,4 +54,4 @@ Except where noted otherwise, released into the [public domain](UNLICENSE).
 Do not just copy, make it better.
 
 
- [1]: https://square.github.io/retrofit/
\ No newline at end of file
+ [1]: https://square.github.io/retrofit/
```

Comment:
Please undo this change. All files should have an empty line at the end to avoid pointless Git changes.

Reply:
I undid the readme change as u suggested. Please merge PR .